spring-boot-project/spring-boot-test-autoconfigure/src/main/java/org/springframework/boot/test/autoconfigure/data/neo4j/DataNeo4jTypeExcludeFilter.java

```diff
@@ -31,4 +44,23 @@
 		super(testClass);
 	}
 
+	private static final Set<Class<?>> DEFAULT_INCLUDES;
+
+	static {
+		Set<Class<?>> includes = new LinkedHashSet<>();
+		includes.add(Driver.class);
+		includes.add(Neo4jClient.class);
+		includes.add(Neo4jTemplate.class);
+		includes.add(Neo4jRepository.class);
+		includes.add(ReactiveNeo4jClient.class);
+		includes.add(ReactiveNeo4jTemplate.class);
+		includes.add(ReactiveNeo4jRepository.class);
+		DEFAULT_INCLUDES = Collections.unmodifiableSet(includes);
```

Comment:
I am a bit confused. Why is this necessary?

Reply:
Basically to avoid collision if a user tries to scan packages containing those classes. If I interpret your comment correctly, maybe too defensive than needed.

Reply:
Yes. If you look all the other exclude type filter implementations we have, none of those do that and that's not the intention anyway.